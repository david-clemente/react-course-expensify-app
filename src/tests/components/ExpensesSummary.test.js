import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should correctly render expensesSummary with one expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={15} expensesTotal={456.55}/>);
    expect(wrapper).toMatchSnapshot();
})

test('should correctly render expensesSummary with multiple expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={2256} expensesTotal={25656.00}/>);
    expect(wrapper).toMatchSnapshot();
})