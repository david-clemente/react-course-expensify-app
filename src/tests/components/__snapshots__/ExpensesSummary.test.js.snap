// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render expensesSummary with multiple expense 1`] = `
<div>
  <h1>
    Viewing 
    2256
     
    expenses
     totalling 
    $256.56
  </h1>
</div>
`;

exports[`should correctly render expensesSummary with one expense 1`] = `
<div>
  <h1>
    Viewing 
    15
     
    expenses
     totalling 
    $4.57
  </h1>
</div>
`;
